src/bundle/Resources/public/js/scripts/helpers/form.error.helper.js

```diff
@@ -1,19 +1,10 @@
 (function (global, doc, ibexa) {
-    const formatLine = (errorMessage) => {
-        const errorIcon = `<svg class="ibexa-icon ibexa-icon--small ibexa-form-error__icon">
-            <use xlink:href="${window.ibexa.helpers.icon.getIconPath('warning-triangle')}"></use>
-        </svg>`;
-        const container = document.createElement('em');
-        const errorMessageNode = document.createTextNode(errorMessage);
-
-        container.classList.add('ibexa-form-error__row');
-        container.insertAdjacentHTML('beforeend', errorIcon);
-        container.append(errorMessageNode);
-
-        return container;
-    };
-
+    // @deprecated, will be removed in 5.0
     ibexa.addConfig('helpers.formError', {
-        formatLine,
+        formatLine: (...args) => {
+            console.warn('helpers.formError.formatLine method is deprecated and will be removed in 5.0');
```

Comment:
`please use ... instead` - when deprecating things you should show how to fix it.